src/components/Header/Header.tsx

```diff
@@ -31,10 +41,10 @@ function Header() {
                 </div>
 
                 <div className={styles.headerPageLinks}>
-                    <a className={combineClasses(styles.headerPageLink, styles.headerPageLinkActive)} href="#about">
+                    <a className={getLinkClassName('about')} href="#about" data-page="about" onClick={onPageLinkClick}>
                         About
                     </a>
-                    <a className={styles.headerPageLink} href="#products">
+                    <a className={getLinkClassName('product')} href="#products" data-page="products" onClick={onPageLinkClick}>
```

Comment:
Yep, and here is the reason for enum: you have a bug with an active state for product link. Here  `productS` there `product`. Whit enum you will save yourself from this kind of bug.